packages/next/client/image.tsx

```diff
@@ -144,7 +152,7 @@ function generateImgAttrs({
   const srcSet = widths
     .map(
       (w, i) =>
-        `${callLoader({ src, quality, width: w })} ${
+        `${callLoader({ src, quality, width: w }, loader)} ${
```

Comment:
It feels kind of weird to add a second loader param to `callLoader` like this. I think you can remove `callLoader` altogether if you unify codepaths of default loaders and custom ones

Reply:
Yep, done.